lib/s3.ex

```diff
@@ -103,7 +103,10 @@ defmodule AntikytheraAws.S3 do
       ]
       args =
         if gzip? do
-          [] = :os.cmd('gzip --stdout #{path} > #{path}.gz') # `gzip` in Amazon Linux (version 1.5 as of 2018/01) does not have "--keep" option
+          # `gzip` in Amazon Linux (version 2 as of 2021/02) does not have "--keep" option
+          {compressed, 0} = System.cmd("gzip", ["--stdout", path])
```

Comment:
I prefer the original code because Erlang VM consumes less memory.
Please let me know if there are any benefits of the new code.

Reply:
As I wrote [here](https://github.com/access-company/antikythera_aws/pull/8#discussion_r582638271), the original code was warned by the credo v1.4.1 in terms of command injection.
Is there another way to avoid the warning keeping memory usage less?

Reply:
I understand. Thanks.

(GNU gzip 1.6 has [`--keep`](http://savannah.gnu.org/forum/forum.php?forum_id=7623) option but our Antikythera instance seems to use v1.5.)